sphinx/troubleshooting.rst

```diff
@@ -127,3 +127,18 @@ MacOS on M1
 
         pipenv run pip uninstall psycopg2-binary
         pipenv run pip install psycopg2-binary --no-cache-dir
+
+
+Webpack Compilation Errors
+==========================
+.. Error::
+
+    | ERROR in /path/to/integreat-cms/src/cms/static/@nodelib/...
```

Comment:
```suggestion
    | ERROR in /path/to/integreat-cms/src/cms/static/\@nodelib/...
```
Somehow, Sphinx thinks that this is a valid email address and converts it into a `mailto`-link :smile: 
So I suggest escaping the `@`...

Reply:
@svenseeberg can you include this change? After that, I think this PR can be merged.